modules/emulator/R/minimize.GP.R

```diff
@@ -195,12 +196,16 @@ mcmc.GP <- function(gp,pckg,x0,nmcmc,rng,format="lin",mix, splinefcns=NULL,
     if(mix == "joint"){
       ## propose new
       xnew <- xcurr
-      for(i in 1:dim){
+      if((g > 2) && ((g - 1) %% settings$assim.batch$jump$adapt == 0)){
+        params.recent <- samp[(g - settings$assim.batch$jump$adapt):(g-1),]
+        colnames(params.recent) <- names(x0)
+        accept.count <- round(jmp@arate[(g-1)/settings$assim.batch$jump$adapt]*100)
```

Comment:
`jmp` is created in this line and has different slots, such as `history`, `arate` etc.
```
jmp <- mvjump(ic=jmp0,rate=ar.target, nc=dim)
```
then it is updated by the `update.jump` function in this line
```
jmp <- update(jmp,samp)
```
so I just wanted to make use of the acceptance rate in there instead of writing another count, but probably these counting and updating should be revised and harmonized all together for the emulator at some point